File: 1/UnitTest/VB/DbManagerTest/OleDb/Access.vb
Imports System
Imports System.Data

Imports NUnit.Framework

Imports Rsdn.Framework.Data

Namespace DbManagerTest.OleDb

    <TestFixture()> _
    Public Class Access
        Inherits VB.DbManagerTest.Test

        Public Overrides ReadOnly Property ConfigurationString() As String
            Get
                Return "Access.OleDb"
            End Get
        End Property

    End Class

End Namespace

File: 1/UnitTest/CS/DbManagerTest/OleDb/Access.cs
using System;
using System.Data;

using NUnit.Framework;

using Rsdn.Framework.Data;

namespace CS.DbManagerTest.OleDb
{
	[TestFixture]
	public class Access : CS.DbManagerTest.Test
	{
		public override string ConfigurationString 
		{
			get 
			{
				return "Access.oledb";
			}
		}

		public override void ExecuteDictionary() {}
		public override void ExecuteDictionary_CommandType_Text() {}
		public override void ExecuteDictionary_Parameters() {}
		public override void ExecuteDictionary_CommandType_Text_Parameters() {}
		public override void ExecutePreparedDictionary() {}
		public override void ExecuteDictionary_Dictionary() {}
		public override void ExecuteDictionary_Dictionary_CommandType_Text() {}
		public override void ExecuteDictionary_Dictionary_Parameters() {}
		public override void ExecuteDictionary_Dictionary_CommandType_Text_Parameters() {}
	}
}
